spec/fixtures/package.json

```diff
@@ -21,8 +21,7 @@
     "noflo-css": "^0.0.2"
   },
   "devDependencies": {
-    "noflo-runtime-iframe": "^0.8.0",
```

Comment:
Does the opener support everything that iframe does, and is it compatible?
We used iframe for running inside Flowhub, the current default dev flow relies on that, as well as the examples?